community/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,123 @@
+addresses, without explicit permission
+ * Other unethical or unprofessional conduct
+ * Advocating for, or encouraging, any of the above behaviors
+
+<br>
+
+## Reporting Guidelines
+
+If you believe someone is violating the code of conduct, please report this in a timely manner (see How to Submit a Report). Code of conduct violations reduce the value of the community for everyone.
+
+The SourceCred community takes reports of misconduct very seriously
+and are committed to preserving and maintaining the welcoming nature of our community.
+All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. All reports will be kept confidential, with the exception of cases where the Benevolent Dictator determines the report should be shared with law enforcement. In those cases, the report will be shared with the proper legal authorities.
+
+In some cases the Benevolent Dictator may determine that a public statement will need to be made. If that’s the case, the identities of all involved parties and reporters will remain confidential unless those individuals instruct us otherwise.
+
+For possibly unintentional breaches of the code of conduct, you may want to respond to the person and point out this code of conduct (either in public or in private, whatever is most appropriate). If you would prefer not to do that, please report the issue (see How to 
+Submit a Report).
+
+### How to Submit a Report
+
+If you feel your safety is in jeopardy or the situation is an emergency, contact local law enforcement before making a report to SourceCred. (In the U.S., dial 911.)
+
+The Benevolent Dictator on behalf of the SourceCred community is committed to promptly addressing any reported issues. If you have experienced or witnessed behavior that violates the SourceCred Code of Conduct, please send an email to concerns@sourcecred.io.
+
+<br>
+
+## License
+
+This code of conduct has been adapted from the USRSE and NumFocus code of conducts, both of which
+were adapted from numerous sources, including:
+
+ - [Geek Feminism wiki](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy) created by the Ada Initiative and other volunteers that is under a [Creative Commons Zero license](https://creativecommons.org/share-your-work/public-domain/cc0/)
+ - the [Contributor Covenant version 1.2.0](http://contributor-covenant.org/version/1/2/0/)
+ - the [Bokeh Code of Conduct](https://github.com/bokeh/bokeh/blob/master/CODE_OF_CONDUCT.md)
+ - the [SciPy Code of Conduct](https://github.com/jupyter/governance/blob/master/conduct/enforcement.md),
+ - the [Carpentries Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html#enforcement-manual)
+ - the [NeurIPS Code of Conduct](https://nips.cc/public/CodeOfConduct).
+
+This Code of Conduct is licensed under a [Creative Commons Attribution 3.0 Unported License](https://creativecommons.org/licenses/by/3.0/).
```

Comment:
Sidenote: I'm not familiar with "Unported" here. Is that about localization? Couldn't find something about that on the page.

Reply:
Yeah my Google searching is for the same page https://creativecommons.org/licenses/by/3.0/ - I was following suit of another license, I couldn't give you specific details about what makes it special (it doesn't seem super special at quick glance).

Reply:
Well the license exists there and I believe we're using it as intended. Not seeing an issue.

If I compare v3 and v4:
- https://creativecommons.org/licenses/by/3.0/
- https://creativecommons.org/licenses/by/4.0/

It becomes: "Attribution 4.0 _International_ (CC BY 4.0)"
Hence my guess it's a localization thing maybe.